server/chat-commands/info.js

```diff
@@ -2339,6 +2339,18 @@ const commands = {
 	},
 	pickrandomhelp: [`/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements.`],
 
+	'!shuffle': true,
+	shuffle(target, room, user) {
+		if (!target) return false;
+		if (!target.includes(',')) return this.parse('/help shuffle');
+		if (!this.runBroadcast(true)) return false;
+		let args = target.split(',');
+		if (args.length < 3) return this.errorReply('Give at least 3 arguments to shuffle.');
```

Comment:
Alternatively, you could make it that if the arguments length is less than 3, parse /pick, idk

Reply:
Why is the a length requirement at all? It seems possible to me to be able to shuffle 0, 1 or 2 elements

Reply:
I can understand 2 elements, but how can you shuffle a list of n < 2 elements? Or, what use case would it fulfill?

Reply:
I support shuffling 2 elements. 0 or 1 should be an error message. It's not like this is a programming API; zero bots should be using it.

Reply:
0 and 1 are noops. why write a partial function when you dont need to?